backend/inspirehep/pidstore/api/authors.py

```diff
@@ -4,7 +4,7 @@
 #
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
-
+from inspirehep.pidstore.minters.bai import BAIMinter
```

Comment:
space between docs :)

Reply:
This is exactly same size of space we have in every test_minters tests. I'm just trying to be consistent.